pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -106,11 +131,15 @@
     <div class="row">
       <main>
 
-        <div class="alert alert-primary alert-dismissible fade show" role="alert" id="pipelineRunning" th:if="${isRunning}">
-          <!-- TODO add progress data too. -->
-          <!-- TODO replace the spinning GIF with something that uses less CPU for rendering. -->
-          <img width="100px" src="../static/Spinning_wheel_throbber_blue.gif" th:src="@{/Spinning_wheel_throbber_blue.gif}">
-          A pipeline is currently running!
+        <div class="alert alert-primary alert-dismissible fade show" role="alert" th:if="${isRunning}">
+          <div class="progress" >
+            <div class="progress-bar progress-bar-striped" style="min-width: 30px; display: none"></div>
```

Comment:
Consider adding a snapshot of this while the pipeline is running, to the PR description.

Reply:
Done